src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.State.cs

```diff
@@ -21,8 +21,9 @@ internal class State
             public SemanticModel Model { get; }
 
             // The members that are not implemented at all.
-            public ImmutableArray<(INamedTypeSymbol type, ImmutableArray<ISymbol> members)> UnimplementedMembers { get; private set; }
+            public ImmutableArray<(INamedTypeSymbol type, ImmutableArray<ISymbol> members)> UnimplementedMembersNotRequiringExplicitImplementation { get; private set; }
```

Comment:
tbh, i find these names confusing now.  How about:

`MembersWithoutExplicitImplementation` and `MembersWithoutExplicitOrImplicitImplementation`?

Reply:
There are 
`MembersWithoutExplicitImplementation`
` MembersWithoutExplicitOrImplicitImplementation`
`MembersWithoutExplicitOrImplicitImplementationWhichCanBeImplicitlyImplemented`

The last seems too long

Reply:
1. MembersWithoutExplicitOrImplicitImplementation could return a tuple with both lists.  I'd be ok with that.
1. MembersWithoutExplicitOrImplicitImplementation could return a list of symbols and a bit saying if it can be implemented implicitly.
3. I actually don't mind the long name :)